index.js

```diff
@@ -1 +1,46 @@
 // Write your solution here!
+const cats = [
+  "Milo", 
+  "Otis", 
+  "Garfield"
+];
+
+function destructivelyAppendCat() {
+  cats.push("Ralph");
+}
+
+function destructivelyPrependCat() {
+  cats.unshift("Bob");
+}
+
+function destructivelyRemoveLastCat() {
+  cats.pop();
+}
+
+function destructivelyRemoveFirstCat() {
+  cats.shift();
+}
+
+function appendCat(){
+  const appendCat = [...cats, "Broom"];
+  return appendCat;
```

Comment:
You could simplify by doing this:
```suggestion
  return [...cats, "Broom"];
```
Same for the others.